Yoast/wordpress-seo
# Open graph article publisher and author

## Summary


* Adds presenters and presentations for Open Graph Article Publisher and Article Author.

This PR can be summarized in the following changelog entry:

* Fixes a bug where the `article:publisher` meta tag would not contain the Person's Facebook account if the Knowledge Graph was set to Person.

## Relevant technical choices:

* I've added helpers for get_post and get_the_author_meta.

## Test instructions

This PR can be tested by following these steps:

* Set Search Appearance > Knowledge Graph to Organization.
* Go to Yoast SEO > Social > Accounts and fill in a Facebook Page URL.
* Navigate to the front end of a post, and see Facebook Page URL in the `article:publisher` meta tag.
* Remove the Facebook Page URL.
* No `article:publisher` meta tag is output.
* Go to the settings of your user (e.g. `admin` and fill in a Facebook account.
* Navigate to the front end of a post written by that user, and see Facebook account URL in the `article:author` meta tag.
* Remove the facebook account of the author. 
* No `article:author` meta tag is output.
* Set Search Appearance > Knowledge Graph to Person. 
* Choose a person to be represented by the site.
* Go to that user's profile and set a Facebook account.
* Navigate to the front end of a post, and see that Facebook account in the `article:publisher` meta tag.

NB: that the user's Facebook account will show up in the `article:publisher` meta tag is new. In the old code, it would incorrectly show the site-wide Facebook Page URL from `Yoast SEO > Social > Accounts`.

## UI changes

## Documentation

## Quality assurance


Fixes #
